generate-skipfile-plan-names

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# vim: set ts=4
+#
+# Copyright 2023-present Linaro Limited
+#
+# SPDX-License-Identifier: MIT
+
+
+import sys
+
+linux_stable_rc_branches = {
+    "linux-4.14.y": "linux-stable-rc",
+    "linux-4.19.y": "linux-stable-rc",
+    "linux-5.4.y": "linux-stable-rc",
+    "linux-5.10.y": "linux-stable-rc",
+    "linux-5.15.y": "linux-stable-rc",
+    "linux-6.1.y": "linux-stable-rc",
+    "linux-6.2.y": "linux-stable-rc",
+}
+
+linux_mainline_next_branches = {
+    "linux-mainline": "master",
+    "linux-next": "master",
+}
```

Comment:
This should not be here... we should be pulled from SQUAD.
However, before that just put them in the lib first.

Reply:
Opened https://github.com/Linaro/squad-client-utils/pull/28 for feedback on draft code for auto detecting branches.